src/afl-fuzz-stats.c

```diff
@@ -384,16 +384,16 @@ void maybe_update_plot_file(afl_state_t *afl, double bitmap_cvg, double eps) {
   /* Fields in the file:
 
      unix_time, afl->cycles_done, cur_path, paths_total, paths_not_fuzzed,
-     favored_not_fuzzed, afl->unique_crashes, afl->unique_hangs, afl->max_depth,
-     execs_per_sec */
+     favored_not_fuzzed, unique_crashes, unique_hangs, max_depth,
+     execs_per_sec, edges_found */
 
   fprintf(
       afl->fsrv.plot_file,
-      "%llu, %llu, %u, %u, %u, %u, %0.02f%%, %llu, %llu, %u, %0.02f, %llu\n",
+      "%llu, %llu, %u, %u, %u, %u, %0.02f%%, %llu, %llu, %u, %0.02f, %llu, %u\n",
```

Comment:
@andreafioraldi maybe also update afl_plot to use this? :)

Reply:
you mean statsd?

Reply:
I mean the afl_plot command. if you write a new entry to the plot file I think it is helpful if the afl_plot command also makes a new graph with that information. you need it anyway I guess (unless you write your own plots)

Reply:
at some point i'll rewrite in python afl plot using seaborn or matplotlib to have nicer and more meaningful graphs

Reply:
also merging multiple afl plot files then please! :)